src/features/snap/components/SnapReport.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState, type FunctionComponent } from 'react';
+
+import { ReportSnapModal } from './modals/ReportSnapModal';
+import { ReportButton } from '../../../components';
+
+type SnapReportProps = {
+  snapName: string;
```

Comment:
you need the snap id too, but i think you can add it in other PR

Reply:
lets do it another PR, i will do that when VC needed.